beets/autotag/mb.py

```diff
@@ -418,7 +418,7 @@ def album_info(release):
         info.media = first_medium.get('format')
 
     genres = release.get('genre-list')
-    if genres:
+    if config['use_mb_genres'] and genres:
```

Comment:
Let's use `config['musicbrainz']['genres']`, so the configuration resides in our existing section for MB options here:
https://github.com/beetbox/beets/blob/627005d4d15622930f36fcaffbc3ad958cd1f6c0/beets/config_default.yaml#L102